llrt_core/src/modules/http/request.rs

```diff
@@ -156,6 +158,13 @@ impl<'js> Request<'js> {
         TypedArray::new(ctx, Vec::<u8>::new()).map(|m| m.into_value())
     }
 
+    async fn blob(&mut self, ctx: Ctx<'js>) -> Result<Blob> {
+        if let Some(bytes) = self.take_bytes(&ctx).await? {
+            return Ok(Blob::from_bytes(bytes, self.content_type.clone()));
```

Comment:
use content-type value from header if this is missing

Reply:
I think I have addressed this by modifying `assign_request()`.

Reply:
Yes and no. It's better to use "on-demand" than eagerly. See my comment :)